Source/DiabloUI/settingsmenu.cpp

```diff
@@ -124,7 +124,7 @@ void UiSettingsMenu()
 		const char *titleText = _("Settings");
 		vecDialog.push_back(std::make_unique<UiArtText>(titleText, MakeSdlRect(PANEL_LEFT, UI_OFFSET_Y + 161, PANEL_WIDTH, 35), UiFlags::FontSize30 | UiFlags::ColorUiSilver | UiFlags::AlignCenter, 8));
 		vecDialog.push_back(std::make_unique<UiScrollbar>(&ArtScrollBarBackground, &ArtScrollBarThumb, &ArtScrollBarArrow, MakeSdlRect(rectList.position.x + rectList.size.width + 5, rectList.position.y, 25, rectList.size.height)));
-		vecDialog.push_back(std::make_unique<UiArtText>(optionDescription, MakeSdlRect(rectDescription), UiFlags::FontSize12 | UiFlags::ColorUiSilverDark | UiFlags::AlignCenter, 1));
+		vecDialog.push_back(std::make_unique<UiArtText>(optionDescription, MakeSdlRect(rectDescription), UiFlags::FontSize12 | UiFlags::ColorUiSilverDark | UiFlags::AlignCenter, 1, IsSmallFontTall() ? 22 : 18));

```

Comment:
Sometimes we display 3 lines of description text.
With height 18 the last line will be near to the border.
To avoid this we could reduce the height to 16 or move the start of the description text up.

## line height 18
![grafik](https://user-images.githubusercontent.com/25415264/143681414-acbf35d3-32a3-48c2-bba8-dc9893afab90.png) 
## line height 16
![grafik](https://user-images.githubusercontent.com/25415264/143681421-f87d56a7-af91-4f58-a669-873887d32fdb.png)

Reply:
I've moved the description up by 4px and it now fits:

![image](https://user-images.githubusercontent.com/216339/143688453-43ceeba8-a3fa-4336-bbb4-e0b6b663f35f.png)